controller/raft/mesh/mesh.go

```diff
@@ -109,9 +116,12 @@ type Mesh interface {
 	// GetOrConnectPeer returns a peer for the given address. If a peer has already been established,
 	// it will be returned, otherwise a new connection will be established
 	GetOrConnectPeer(address string, timeout time.Duration) (*Peer, error)
+	IsReadOnly() bool
 }
 
-func New(id *identity.TokenId, raftId raft.ServerID, raftAddr raft.ServerAddress, bindHandler channel.BindHandler) Mesh {
+func New(id *identity.TokenId, version string, raftId raft.ServerID, raftAddr raft.ServerAddress, bindHandler channel.BindHandler) Mesh {
```

Comment:
I think you can just embed a non-pointer atomic.Bool in the impl. It won't need to be initialized b/c the default value will be false.

Reply:
changed to non-pointer